.github/workflows/publish.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     - name: Build and push image
       uses: docker/build-push-action@v2
       with:
-        tags: 'ghcr.io/fossas/haskell-dev-tools:9.0.2'
+        tags: 'ghcr.io/fossas/haskell-dev-tools:9.4.7'
```

Comment:
[no action] Unfortunate that we can't communicate the breaking change through our version, since we just reuse GHC versions as the version for this image :(

Reply:
Yes, I agree. If you want I could try to put some time into having versions like `9.4.7-1`, `9.4.7-2` and so on. But adding a Changelog was my band-aid fix.

I am hoping that in the future I'll be more up on maintaining these, but I'm not sure what that will look like in practice yet.

Reply:
Yeah, I don't think it's worth the time, especially since I think we're the only users